src/expression-engine/models/ee/model.js

```diff
@@ -17,12 +17,16 @@ export class EE extends Heighliner {
   contentImages(markup) {
     if (!markup) return [];
 
-    let images = markup.match(/src=".*\.(jpg|jpeg|png)"/gmi);
+    const images = markup.match(/src=".*\.(jpg|jpeg|png)"/gim);
```

Comment:
@redreceipt Any idea about this change here? It's in a couple of other places too. Mostly concerned with the `/gmi` and `/gim` change. Do you know what this is?

Reply:
that's a regex match function, those are flags that the order doesn't matter, guessing prettier just sorts them

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#Advanced_searching_with_flags_2

Reply:
Haha! They alphabetized them. 😂